swarm/storage/ldbstore.go

```diff
@@ -288,13 +317,8 @@ func (s *LDBStore) collectGarbage(ratio float32) {
 		gcnt++
 	}
 
-	sort.Slice(garbage[:gcnt], func(i, j int) bool { return garbage[i].value < garbage[j].value })
-
-	cutoff := int(float32(gcnt) * ratio)
-	metrics.GetOrRegisterCounter("ldbstore.collectgarbage.delete", nil).Inc(int64(cutoff))
-
-	for i := 0; i < cutoff; i++ {
-		s.delete(garbage[i].idx, garbage[i].idxKey, garbage[i].po)
+	for _, garbageItem := range garbage {
+		s.delete(garbageItem.idx, garbageItem.idxKey, garbageItem.po)
```

Comment:
we should just do the deletes in one batch in one loop and write it in one go, or do the deletes the same way as writes,